Components/Sources/Components/Components/Suggested Edits/Image Recommendations/WKImageRecommendationBottomSheetView.swift

```diff
@@ -0,0 +1,383 @@
+        imageWidthConstraint.priority = .required
+        let imageHeightConstraint = imageView.heightAnchor.constraint(equalToConstant: imageViewHeight)
+        imageHeightConstraint.priority = .required
+
+        NSLayoutConstraint.activate([
+            linkButtonTopConstarint,
+            imageWidthConstraint,
+            imageHeightConstraint
+        ])
+
+        setupTextViewExclusionPath()
+        updateColors()
+        setupToolbar()
+    }
+
+    private func setupTextViewExclusionPath() {
+        let height = regularSizeClass ? imageViewHeight - buttonHeight : (imageViewHeight - padding) - buttonHeight
+        let rectangleWidth: CGFloat = cutoutWidth
+        let rectangleHeight: CGFloat = height
+
+        let layoutDirection = textView.effectiveUserInterfaceLayoutDirection
+        let isRTL = layoutDirection == .rightToLeft
+
+        let rectangleOriginX: CGFloat
+        if isRTL {
+            let width = self.frame.width
+            rectangleOriginX = width - rectangleWidth - textView.textContainerInset.right - padding * 2.5
+        } else {
+            rectangleOriginX = textView.textContainerInset.left
+        }
+
+        let rectangleOriginY: CGFloat = textView.textContainerInset.top
+
+        let rectangleFrame = CGRect(x: rectangleOriginX, y: rectangleOriginY, width: rectangleWidth, height: rectangleHeight)
+        let rectanglePath = UIBezierPath(rect: rectangleFrame)
+
+        textView.textContainer.exclusionPaths = [rectanglePath]
+    }
+
+    private func updateColors() {
+        backgroundColor = theme.paperBackground
+        textView.textColor = theme.secondaryText
+        titleLabel.textColor = theme.text
+        imageLinkButton.setTitleColor(theme.link, for: .normal)
+        iconImageView.tintColor = theme.link
+        toolbar.barTintColor = theme.midBackground
+    }
+
+    private func configure() {
+        imageView.image = viewModel.imageThumbnail
+        textView.text = viewModel.imageDescription
+        titleLabel.text = viewModel.headerTitle
+        imageLinkButton.setAttributedTitle(getImageLinkButtonTitle(), for: .normal)
+    }
+
+    private func getImageLinkButtonTitle() -> NSMutableAttributedString {
+        let attributedString = NSMutableAttributedString()
+        if let imageAttachment = WKIcon.externalLink {
+            let attachment = NSTextAttachment(image: imageAttachment)
+            attributedString.append(NSAttributedString(string: viewModel.imageTitle))
+            attributedString.append(NSAttributedString(string: "  "))
+            attributedString.append(NSAttributedString(attachment: attachment))
+        }
+        return attributedString
+    }
+
+    private func setupToolbar() {
+        let spacer = UIBarButtonItem(systemItem: .flexibleSpace)
+        toolbar.setItems([yesToolbarButton, spacer, noToolbarButton, spacer, notSureToolbarButton], animated: true)
```

Comment:
I see some constraint errors in the console when these are set, but so far I haven't figured out what's causing it.